### DISCUSSION TITLE
Increase the length of the `PurposeOfLoan` field

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5972] [Feature Requests]
We should increase the `PurposeOfLoan` field in the `loan`table to at least 128 characters.

From Jordi at MCNB on [Trello](https://trello.com/c/Irpyjgku):
> The field purposeOfLoan in the Loan table has 64 chararter leght, is it possible to expand the field al least 128 chararters?
>
>We usually put the name of the exhibition in this field when it is a cultural loan, and with 64 characters it is a bit limited. Otherwise, we will use the remarks field.

**Comment by g1rly-c0d3r** [comment]
Linked in https://github.com/specify/specify7/issues/8186
